modules/ROOT/pages/sysconfig-network-configuration.adoc

```diff
@@ -0,0 +1,966 @@
+   arguments added, (i.e., `coreos.inst.install_dev=`), you can also
+   append networking arguments there and they will apply to the install boot and
+   also the first boot (Ignition boot) of the installed machine.
+
+3. For a PXE boot you can add networking kernel arguments to your
+   existing set of kernel arguments in your PXE configuration.
+
+An example set of kernel arguments for statically configuring an IP
+address for `ens2` looks like like:
+
+[source, bash]
+----
+ip=10.10.10.10::10.10.10.1:255.255.255.0:myhostname:ens2:none:8.8.8.8
+----
+
+The syntax is a bit hard to work with. An easy way to work with
+it is to write a small script that will fill in the items for you.
+For the example above, something like this should work:
+
+[source, bash]
+----
+ip='10.10.10.10'
+gateway='10.10.10.1'
+netmask='255.255.255.0'
+hostname='myhostname'
+interface='ens2'
+nameserver='8.8.8.8'
+echo "ip=${ip}::${gateway}:${netmask}:${hostname}:${interface}:none:${nameserver}"
+----
+
+
+==== via `coreos-installer --copy-network`
+
+For manual bare metal install workflows it may not be preferable to use
+dracut kernel arguments for configuring network:
+
+- the syntax is not very user friendly
+- manipulating kernel arguments by grabbing the grub prompt can be challenging
+
+The `--copy-network` option to the `coreos-installer` will copy the
+files from `/etc/NetworkManager/system-connections/` directory into
+the installed system. For an interactive install this allows the user
+to populate networking configuration in a variety of ways before doing
+the install:
+
+- using the `nmcli` command
+- using the `nmtui` TUI interface
+- writing files directly
+- using another tool of choice
+
+It also allows the user to do hardware discovery on the node (i.e.
+"what are my interface names?"). For an example of this workflow
+see https://[this demo].
+
+
+==== via Afterburn
+
+On certain platforms Afterburn will inject networking configuration,
+either configured by the user or by the platform, during the initramfs.
+This is primarily done right now for VMWare and allows for users to
+specify networking configuration in the form of dracut networking
+arguments without having to stop the boot of the machine and manually
+inject those arguemnts themselves.
+
+See 
+https://github.com/coreos/afterburn/blob/master/docs/usage/initrd-network-cmdline.md[the Afterburn documentation]
+for more information.
+
+==== via Ignition
+
+Networking configuration can be performed by writing out files
+described in an Ignition config. These are 
+https://developer.gnome.org/NetworkManager/stable/nm-settings-keyfile.html[NetworkManager keyfiles]
+that are written to `/etc/NetworkManager/system-connections/` that
+tell NetworkManager what to do.
+
+Any configuration provided via Ignition will generally take precedence
+over any other method of configuring the Network for a Fedora CoreOS
+instance.
+
+An example FCCT configuration for the same static networking example
```

Comment:
Maybe a link to our FCCT docs - https://docs.fedoraproject.org/en-US/fedora-coreos/producing-ign/ ?